recipes/kraken2/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
     - Makefile.patch
 
 build:
-  number: 1
+  number: 2
   has_prefix_files:
     - libexec/kraken2
```

Comment:
You can remove the lines 17-20 which triggers the conda-build bug. It's not needed as prefix replacement is done by default

Reply:
Thanks a lot @isuruf, that worked !